packages/dso-toolkit/components/Componenten/application-heading/application-heading.njk

```diff
@@ -0,0 +1,15 @@
+<div class="dso-app-heading">
+  {% if title %}
+    <h1>{{ title }}</h1>
+  {% endif %}
+  {% if subtitle %}
+    <h2 class="dso-steps-indicator">
```

Comment:
wat nou dus echt heel stom is, is als je een dso-app-heading hebt met een h1 en een h2 maar GEEN steps. Dan is het toch hardstikke stom dat die h2 de class 'dso-steps-indicator' zou moeten hebben?

ik zou zeggen, if subtitle maar geen steps, dan is de h2 gewoon een h2